traffic_ops/app/t/api/1.2/deliveryservice.t

```diff
@@ -30,302 +30,407 @@ use Test::TestHelper;
-    ->json_is( "/response/xmlId" => "ds_1")
-    ->json_is( "/response/displayName" => "ds_displayname_11")
-    ->json_is( "/response/orgServerFqdn" => "http://10.75.168.91")
-    ->json_is( "/response/cdnName" => "cdn1")
-    ->json_is( "/response/profileName" => "CCR1")
-    ->json_is( "/response/protocol" => "2")
-    ->json_is( "/response/type" => "HTTP")
-    ->json_is( "/response/multiSiteOrigin" => "0")
-    ->json_is( "/response/regionalGeoBlocking" => "0")
-    ->json_is( "/response/active" => "true")
-    ->json_is( "/response/matchList/0/type" => "HOST_REGEXP")
-    ->json_is( "/response/matchList/0/setNumber" => "0")
-    ->json_is( "/response/matchList/0/pattern" => ".*\\.my_vod1\\..*")
-            , 'Does the deliveryservice details return?';
-
-
-# It allows updating a delivery service in a CDN with a matching regex in another CDN's delivery service
-
-$ds_id = &get_ds_id('ds_3');
-
-ok $t->put_ok('/api/1.2/deliveryservices/' . $ds_id . '/update' => {Accept => 'application/json'} => json => {
-		"xmlId" => "ds_3",
-		"displayName" => "ds_displayname_1",
-		"protocol" => "1",
-		"orgServerFqdn" => "http://10.75.168.91",
-		"cdnName" => "cdn2",
-		"profileName" => "CCR2",
-		"type" => "HTTP",
-		"multiSiteOrigin" => "0",
-		"regionalGeoBlocking" => "1",
-		"active" => "false",
-		"matchList" => [
-			{
-				"type" =>  "HOST_REGEXP",
-				"setNumber" =>  "0",
-				"pattern" => ".*\\.ds_1\\..*"
-			},
-			{
-				"type" =>  "HOST_REGEXP",
-				"setNumber" =>  "1",
-				"pattern" => ".*\\.my_vod1\\..*"
-			}
-		]})
-	->status_is(200)->or( sub { diag $t->tx->res->content->asset->{content}; } )
-		->status_is(200)->or( sub { diag $t->tx->res->content->asset->{content}; } )
-		->json_is( "/response/xmlId" => "ds_3")
-		->json_is( "/response/displayName" => "ds_displayname_1")
-		->json_is( "/response/orgServerFqdn" => "http://10.75.168.91")
-		->json_is( "/response/cdnName" => "cdn2")
-		->json_is( "/response/profileName" => "CCR2")
-		->json_is( "/response/protocol" => "1")
-		->json_is( "/response/type" => "HTTP")
-		->json_is( "/response/multiSiteOrigin" => "0")
-		->json_is( "/response/regionalGeoBlocking" => "1")
-		->json_is( "/response/active" => "false")
-		->json_is( "/response/matchList/0/type" => "HOST_REGEXP")
-		->json_is( "/response/matchList/0/setNumber" => "0")
-		->json_is( "/response/matchList/0/pattern" => ".*\\.ds_1\\..*")
-		->json_is( "/response/matchList/1/type" => "HOST_REGEXP")
-		->json_is( "/response/matchList/1/setNumber" => "1")
-		->json_is( "/response/matchList/1/pattern" => ".*\\.my_vod1\\..*")
-	, 'Does the deliveryservice details return?';
-
-# It does not allow changing a regex to be the same as an existing one in another DS in the same CDN
-$ds_id = &get_ds_id('ds_1');
-ok $t->put_ok('/api/1.2/deliveryservices/' . $ds_id . '/update'  => {Accept => 'application/json'} => json => {
-			"xmlId" => "ds_1",
-			"displayName" => "ds_displayname_2",
+sub run_ut {
```

Comment:
Is there a reason you put everything inside run_ut(). I don't think it is done this way in other places. And by doing this, I can't really tell what you've changed in this file as it looks like you changed everything.